src/Person.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Person.scss';
+
+export function Person({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+
+      <p className="Person__partner">
+        {isMarried ? (
+          `My ${sex === 'f' ? 'husband' : 'wife'}'s name is ${partnerName}`
```

Comment:
It's better to create a separate variable before return statements where you can store the result of inner checking (sex === 'f')
```suggestion
          `My ${partner}'s name is ${partnerName}`
```